yarn.config.cjs

```diff
@@ -93,14 +93,9 @@ module.exports = defineConfig({
         // All non-root packages must have a license, defaulting to MIT.
         await expectWorkspaceLicense(workspace);
 
-        if (
-          workspace.ident !== '@metamask/design-system-react-native' &&
-          workspace.ident !== '@metamask/design-system-twrnc-preset'
-        ) {
-          // All non-root packages must set up ESM- and CommonJS-compatible
-          // exports correctly.
-          expectCorrectWorkspaceExports(workspace);
-        }
+        // All non-root packages must set up ESM- and CommonJS-compatible
+        // exports correctly.
+        expectCorrectWorkspaceExports(workspace);
```

Comment:
Removed special-case export check for '@metamask/design-system-react-native' and '@metamask/design-system-twrnc-preset' since these packages are now fully aligned with the monorepo standards. No longer need to exclude them from export verification.